ui/admin/app/styles/app.scss

```diff
@@ -333,14 +333,16 @@
   margin-bottom: sizing.rems(xl);
   padding: sizing.rems(l);
 
-  > div {
+  > div,
+  > p:not(:last-of-type) {
```

Comment:
I understand this is for the `Hds::Text::Body` component, but we have a ticket to add margin in a common stylesheet, so we are not adding margin wherever we use this component. I'm inclined towards using `<p>` for this until we have the style changes in a common stylesheet; that way, it is easier to replace the `p` tag instead of tracking down the styles.

Reply:
Sounds like a plan! I reverted the `Hds::Text::Body` conversion and this additional class then. If you're happy with the current result feel free to merge – I'm not authorized to merge, which is fine.